src/logics/compound-v3/tokens/data/polygon.json

```diff
@@ -40,5 +47,19 @@
     "decimals": 18,
     "symbol": "WMATIC",
     "name": "Wrapped Matic"
+  },
+  "MaticX": {
+    "chainId": 137,
+    "address": "0xfa68FB4628DFF1028CFEc22b4162FCcd0d45efb6",
+    "decimals": 18,
+    "symbol": "MaticX",
+    "name": "Liquid Staking Matic (PoS)"
+  },
+  "stMATIC": {
```

Comment:
I didn't find this token in[ compound-v3 config](https://github.com/compound-finance/comet/blob/main/deployments/polygon/usdc/configuration.json). Where would we use this token for?

Reply:
Removed. rETH is a collateral token built from refresh-tokens-data.ts
I think their doc is out of date, you could find it on https://app.compound.finance/?market=usdc.e-polygon and onchain data